katalog/curator-s3/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM elasticsearch:6.4.1
```

Comment:
We need to create this container image using the current es version of our stack, 7.X

Reply:
i know, was only an example :)